extensions/amp-experiment/0.1/amp-experiment.js

```diff
@@ -46,12 +47,12 @@ export class AmpExperiment extends AMP.BaseElement {
         Object.keys(config).map(experimentName => {
           return allocateVariant(this.getWin(), config[experimentName])
               .then(variantName => {
-                if (variantName) {
-                  results[experimentName] = variantName;
-                }
+                results[experimentName] = variantName;
```

Comment:
Nope, we're inside a `Promise.all` here, so we'd get an array of `[results, results, results, ...]`.